test/core/event_engine/posix/BUILD

```diff
@@ -191,3 +194,26 @@ grpc_cc_test(
         "//test/core/util:grpc_test_util",
     ],
 )
+
+grpc_cc_test(
+    name = "posix_event_engine_connect_test",
+    srcs = ["posix_event_engine_connect_test.cc"],
+    external_deps = ["gtest"],
+    language = "C++",
+    tags = [
+        "no_windows",
+    ],
+    uses_event_engine = True,
+    uses_polling = True,
```

Comment:
Does this test need to be replicated for every poller using the `@poller=X` test duplication mechanism?

Reply:
Yes the event engine can instantiate both types of pollers. So I think its good to test connection cancellation/timeouts with both pollers